docs/api/grpc.md

```diff
@@ -8,12 +8,12 @@ TODO
 
 ## api.proto
 
-[pkg/gadget-service/api/api.proto](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadget-service/api/api.proto)
-
 TODO
 
+[pkg/gadget-service/api/api.proto](https://github.com/inspektor-gadget/inspektor-gadget/blob/%IG_BRANCH%/pkg/gadget-service/api/api.proto)
```

Comment:
This one took a while. The action we're using for checking the broken links just ignore it because it was not able to parse it. I added some bash logic to replace those constants before running the "broken link action".